internal/access/resource_access_workflow.go

```diff
@@ -104,7 +104,7 @@ func (r *AccessWorkflowResource) Schema(ctx context.Context, req resource.Schema
 			},
 			"try_extend_after_seconds": schema.Int64Attribute{
 				MarkdownDescription: "(Deprecated) The amount of time after access is activated that extending access can be attempted. As a starting point we recommend setting this to half of the `access_duration_seconds`.",
-				Required:            true,
+				Optional:            true,
```

Comment:
Think you can add a deprecation message 
https://developer.hashicorp.com/terraform/plugin/framework/handling-data/schemas#deprecationmessage
```suggestion
				Optional:            true,
				DeprecationMessage: "This field is no longer supported. Use extension_conditions to configure workflow extensions"
```